### PR TITLE
Added an `lambdas.Assert()` as supplement of `lambdas.Must()` for cases function only return an error

### PR DESCRIPTION
**Pull request by kuchaguangjie**
1. update `.gitignore` to ignore editor like `goland` and `vscode`, and temp files on linux.
2. Added an `lambdas.Assert()` as supplement of `lambdas.Must()` for cases function only return an error.

### CODE REVIEW

**Comment by kuchaguangjie** [comment]
If you have any suggestion on naming the method, please share, I was also thinking about the name `MustVoid` ..

**Comment by orsinium** [comment]
Good idea! And you're right, I'm also not sure if this is a good name. For me, assert would be something like this:

```
func Assert(test bool) {
	if !test {
		panic("oh no")
	}
}
```

So, let's think together a bit about a better name. What about `Ensure`? 🤔

**Comment by kuchaguangjie** [comment]
The new method is kinda of a `overload` version of the existing `Must()` function, so maybe `MustXxx()` makes them more related, e.g `MustWithoutReturnValue()`, but that's too long .. , or `MustEnsure()` ?
I think `Ensure()` also works.

**Comment by orsinium** [comment]
Good thinking. Let's call it `Ensure` for now, and I'll consider aliasing later both of the functions with the arity suffix, like `Must1`, `Must2`, etc.

Please, update also the readme, it currently lists all functions: https://github.com/life4/genesis/tree/master/lambdas.

File: lambdas/README.md [hunk 1]
```diff
@@ -7,22 +7,23 @@ These functions are especially helpful in combination with other `genesis` packa
...
+12. IsNil
+13. IsNotNil
+14. Must
+15. Ensure
+16. Safe
+17. DefaultTo
+18. Abs
+19. Min
+20. Max
+21. Default
```
Comment (orsinium) [inline lambdas/README.md:29]: Don't touch my ~fridge~ ones!

**Comment by orsinium** [comment]
Thank you :) I'll build a release shortly, you'll get a notification.

**Pull request merged**